source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -909,6 +909,10 @@ def _initialize_impl(self):
         # -- articulation
         self._root_physx_view = self._physics_sim_view.create_articulation_view(root_prim_path_expr.replace(".*", "*"))
 
+        # check if the articulation was created
+        if self._root_physx_view._backend is None:
+            raise RuntimeError(f"Failed to create articulation at: {self.cfg.prim_path}. Please check PhysX logs.")
```

Comment:
NIT: Can you add this and the other raised Exceptions to the `Raises:` section of docstring. Not critical by any means!